shared/src/commonTest/kotlin/co/nimblehq/mark/kmmic/data/repository/AuthRepositoryTest.kt

```diff
@@ -0,0 +1,106 @@
+package co.nimblehq.mark.kmmic.data.repository
+
+import app.cash.turbine.test
+import co.nimblehq.mark.kmmic.data.service.auth.AuthService
+import co.nimblehq.mark.kmmic.data.service.auth.model.AuthTokenApiModel
+import co.nimblehq.mark.kmmic.data.service.token.TokenService
+import co.nimblehq.mark.kmmic.domain.model.AuthToken
+import co.nimblehq.mark.kmmic.domain.repository.AuthRepository
+import kotlin.test.BeforeTest
+import io.kotest.matchers.shouldBe
+import io.mockative.*
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.first
+import kotlinx.coroutines.flow.flow
+import kotlinx.coroutines.flow.flowOf
+import org.koin.core.context.startKoin
+import org.koin.dsl.module
+import kotlin.test.Test
+import kotlinx.coroutines.test.runTest
+import org.koin.core.context.stopKoin
+import kotlin.test.AfterTest
+
+@ExperimentalCoroutinesApi
+class AuthRepositoryTest {
+    @Mock
+    private val mockAuthService = mock(classOf<AuthService>())
+    @Mock
+    private val mockTokenService = mock(classOf<TokenService>())
+    private lateinit var authRepository: AuthRepository
+    private val mockTokenApi = AuthTokenApiModel(
+        "id",
+        "type",
+        "accessToken",
+        "tokenType",
+        1,
+        "refreshToken",
+        1
+    )
+    private val mockToken = AuthToken(mockTokenApi)
+    private val mockThrowable = Throwable("mock")
```

Comment:
Optional: some line breaks between groups would be nice.
```suggestion
    
    @Mock
    private val mockAuthService = mock(classOf<AuthService>())
    @Mock
    private val mockTokenService = mock(classOf<TokenService>())
    
    private lateinit var authRepository: AuthRepository
    
    private val mockTokenApi = AuthTokenApiModel(
        "id",
        "type",
        "accessToken",
        "tokenType",
        1,
        "refreshToken",
        1
    )
    private val mockToken = AuthToken(mockTokenApi)
    private val mockThrowable = Throwable("mock")
```

Reply:
@AVI5HEK updated at [560719c](https://github.com/markgravity/kmm-ic/pull/110/commits/560719c698c53127c05b016497d6ef08c4d5b098)